src/platform/nm-linux-platform.c

```diff
@@ -5783,6 +5783,43 @@ link_vxlan_add (NMPlatform *platform,
 	g_return_val_if_reached (FALSE);
 }
 
+static gboolean
+link_6lowpan_add (NMPlatform *platform,
+                  const char *name,
+                  int parent,
+                  const NMPlatformLink **out_link)
+{
+	nm_auto_nlmsg struct nl_msg *nlmsg = NULL;
+	struct nlattr *info;
+
+	_LOGD ("adding 6lowpan '%s' parent %u", name, parent);
```

Comment:
this results in duplicate logging lines. See "platform: avoid double log messages" commit on bg/sriov-vf-rh1555013 branch.

Reply:
Fixed.